src/views/ApplicationViews.js

```diff
@@ -35,7 +36,8 @@ export const ApplicationViews = () => {
         <Route path="tags" element={<Tags />} />
         <Route path="new-tag" element={<NewTagForm />} />
         <Route path="logout" element={<Logout />} />
+        <Route path="create-post" element={<CreatePostForm />} />
```

Comment:
To use the currentUser prop in CreatePostForm, you have to pass it in on this line, too.
e.g. element={<CreatePostForm currentUser={currentUser} />}